snap/quota/quota.go

```diff
@@ -142,6 +165,15 @@ func (grp *Group) GetQuotaResources() Resources {
 	if grp.TaskLimit != 0 {
 		resourcesBuilder.WithThreadLimit(grp.TaskLimit)
 	}
+	if grp.JournalLimit != nil {
+		resourcesBuilder.WithJournalNamespace()
+		if grp.JournalLimit.Size != 0 {
+			resourcesBuilder.WithJournalSize(grp.JournalLimit.Size)
+		}
+		if grp.JournalLimit.RateCount != 0 && grp.JournalLimit.RatePeriod != 0 {
+			resourcesBuilder.WithJournalRate(grp.JournalLimit.RateCount, grp.JournalLimit.RatePeriod)
+		}
+	}
```

Comment:
Can w test it, or do you have tests in another PR?

Reply:
I have added a unit test for this now, thanks for catching this!